WooCommerce/Classes/ViewRelated/Dashboard/Onboarding/StoreOnboardingTaskViewModel.swift

```diff
@@ -27,7 +27,7 @@ struct StoreOnboardingTaskViewModel: Identifiable, Equatable {
             icon = .domainsImage
             title = Localication.CustomizeDomains.title
             subtitle = Localication.CustomizeDomains.subtitle
-        case .payments:
+        case .payments, .woocommercePayments:
```

Comment:
❓  in case a site has both `payments` and `woocommercePayments` tasks, do we have a way to only show one?

Reply:
I had the same question, and I asked Ghidora about this earlier. They confirmed that both `payments` and `woocommercePayments` won't be present simultaneously in the JSON response. 

Internal discussion in which I raised this question - p1677835903386069/1677834025.356109-slack-C01SFMVEYAK